### ISSUE TITLE
Enter a short summary here!

### ISSUE THREAD

**Issue by Spireons** [ValveSoftware/Dota2-Gameplay#2548]
### Item name

 Morph ult (r)  with shard vs Face Void 

### Description

When you use ult with shard you get improvements  even if the enemy does not have shard but vs faceless void game think both of you must have shard , pls fix this shit =) 

### Example Match ID and Timestamp

6733732596 all game

**Comment by Mabadman** [comment]
This is intended behaviour. 
When Morphling or Rubick cast a spell, the spell will be upgraded if they themselves have that upgrade item (scepter or shard).

**Comment by Spireons** [comment]
> This is intended behaviour. When Morphling or Rubick cast a spell, the spell will be upgraded if they themselves have that upgrade item (scepter or shard).
 Yes , but vs Faceless Void it isnt wrok on Morph , you can test in lobby

**Comment by TheUnholyWrath** [comment]
![20220901045312_1](https://user-images.githubusercontent.com/46859977/187759747-8fda99bf-1960-400b-9d44-649237a5f73b.jpg)

**Comment by Spireons** [comment]
> TheUnholyWrath
![20220831215856_1](https://user-images.githubusercontent.com/112565254/187761249-26b9c09b-0998-4adf-bf82-1ba758b7f236.jpg)
are you dumb ?

**Comment by DarkoniusXNG** [comment]
> This is intended behaviour. When Morphling or Rubick cast a spell, the spell will be upgraded if they themselves have that upgrade item (scepter or shard).

Spireons is trying to say that: Morphling (morphed into Faceless) gets shard buff and non working reverse time walk (useless button) when only Faceless Void has a bought shard. Morphling before morphing doesnt have shard.
So definitely not intended. It's just hard to explain. And his broken english is confusing you even more.

**Comment by Mabadman** [comment]
> > This is intended behaviour. When Morphling or Rubick cast a spell, the spell will be upgraded if they themselves have that upgrade item (scepter or shard).
> 
> Spireons is trying to say that: Morphling (morphed into Faceless) gets shard buff and non working reverse time walk (useless button) when only Faceless Void has a bought shard. Morphling before morphing doesnt have shard. So definitely not intended. It's just hard to explain. And his broken english is confusing you even more.

I see, thanks for explaining. I've rescinded my downvote.

**Comment by Spireons** [comment]
> > This is intended behaviour. When Morphling or Rubick cast a spell, the spell will be upgraded if they themselves have that upgrade item (scepter or shard).
> 
> Spireons is trying to say that: Morphling (morphed into Faceless) gets shard buff and non working reverse time walk (useless button) when only Faceless Void has a bought shard. Morphling before morphing doesnt have shard. So definitely not intended. It's just hard to explain. And his broken english is confusing you even more.

Yes , my english isnt so good , but you got it and for hard to explain correctly
